src/helpers/images.js

```diff
@@ -9,9 +9,10 @@ const BasicImage = (src, alt) => {
   return (
     <Img
       style={{
-        width: "100%",
+        width: "75%",
+        maxWidth: "280px",
         height: "auto",
-        borderRadius: "8px",
+        borderRadius: "100%",
```

Comment:
Make sure that these changes will not affect badly on Img tags that are used in other components in UI/UX wise.